### PR TITLE
For demo, modify Page layout, Locations Base

### PR DESCRIPTION
**Pull request by devjk1**
- remove references to _$page->location_ and _$child_page->market_
  - remove include page/location/partials/_identity_tag
  - remove placeholder content
- add DomesticAddress & Phone livewire components to save both to db via Pages (using this layout)

### CODE REVIEW

**Comment by joshtorres** [comment]
thanks @devjk1. @drewroberts is this what you had in mind? I can jump on a call and go over it if you'd like.

File: resources/views/page/location/base.blade.php [hunk 1]
```diff
@@ -1,14 +1,18 @@
 @extends('support::base')
 
 @section('content')
-    @include('locations::page.location.partials._identity_tag')
```
Comment (pdbreen) [inline resources/views/page/location/base.blade.php:4]: Please explain -- why are you removing this?   It is required to satisfy unit testing validation.

**Comment by pdbreen** [comment]
Why???  Makes no sense... These must exist

> remove references to $page->location and $child_page->market

**Comment by devjk1** [comment]
@pdbreen Hi Patrick.  When I create a Page using the Locations Base layout, hese are null $page->location and $child_page->market
In the pics below, when creating a Page using the Locations Base layout, there is no option to set a location to this page; there is no option to set child_page or market.  I'm not sure how we should pull these relationships, a little unfamiliar with this part of the app.

image
image

**Comment by pdbreen** [comment]
@devjk1 this is the locations package - not the blog package.  All locations and markets have a page association, meaning the inverse always exists, so the identity tag should remain.  

If you are testing from Nova admin, create a market or location and then access the route.  Please make sure you understand what you are doing before deleting code someone else put there for a reason.

**Comment by pdbreen** [comment]
@devjk1 When creating a page, you should NOT use the location layout.  You must use the page layout.

**Comment by devjk1** [comment]
@pdbreen Sorry about that, you're right.  I will close this PR.

**Comment by joshtorres** [comment]
@pdbreen this is my fault. I obviously misunderstood how to use these layouts. Can you meet sometime tomorrow to help me understand the proper way for these front end views to be created? Or I can read through documentation if we have any. We've been trying to figure out where to add these address and phone components on the front end and I told Jinsik to put them there.

**Comment by pdbreen** [comment]
@joshtorres not available - but answer is pretty simple.  When creating a blog page, use the blog page layout, not the location page layout.  The blog page layouts (both base and amp) are defined in the blog package.  When creating locations (or markets), a page gets created automatically with the proper location page layout assigned.  For layout modification, all was fine except for removal of the identity tag include.  There should never be a reason to remove the identity tag include from any of the layouts that have them (this concept exists in multiple packages to facilitate testing).  Drew should be able to answer any other questions.

**Comment by joshtorres** [comment]
Ok, thank you. I do see what you mean about the tag. However, we aren't trying to make a blog page, we are trying to test the address and phone component. Also, we can't just create a page in Nova as these are actual fields we need,unless I'm misunderstanding that too. Do you know what page or layout we should use for that? If not, I will follow up with Drew again. Thanks again for your time.

**Comment by pdbreen** [comment]
Define a custom test route / controller / blade component directly in the demo app.  All the stuff you're messing around in are the generic foundation layouts for other things.

**Comment by joshtorres** [comment]
Got it thank you

**Pull request closed without merging**